static/js/logs.js

```diff
@@ -15,9 +15,9 @@ const renderLogs = (logLines) => {
 
 const renderNav = (pageN, maxPage) => {
   const paginationNode = document.getElementById('pagination');
-  paginationNode.innerHTML = ''
+  paginationNode.innerHTML = '';
 
-  if (maxPage == 1 || maxPage == 0) {
+  if (maxPage <= 1) {
     return; // Don't display pagination on log files that fit in a single page.
   }
   const root = document.createElement('div');
```

Comment:
Move the buttons in the HTML if they are always rendered. Add an id (previous-page, next-page) to attach the event listener. Make them invisible by default.